### DISCUSSION TITLE
autosuggest + autocomplete (switch between?)

### DISCUSSION THREAD

**Discussion by ghostsquad** [marlonrichert/zsh-autocomplete#534] [Q&A] [answered]
I'm little bit confused about how I switch between the suggested filenames and the history entry available in this screenshot below.

![image](https://user-images.githubusercontent.com/117093144/210657305-7a5c9de1-5a3e-41d8-a060-e84eddb96124.png)

If I press [right arrow] it uses the entire _command_, but what I really want to do is select one of the files available. If this isn't the right place to post this question, please let me know.

**Comment by marlonrichert** [comment, marked as answer]
> If I press [right arrow] it uses the entire command, but what I really want to do is select one of the files available.

That’s what [zsh-users/zsh-autosuggestions](https://github.com/zsh-users/zsh-autosuggestions) does by default.

To insert a completion from the list, either press Tab to insert the first one or use Down Arrow to enter menu selection.
